spec/lib/sufia/user_stat_importer_spec.rb

```diff
@@ -164,6 +165,13 @@
 
       expect(frodos_stats[0].file_downloads).to eq(frodo_file_1_download_stats[0].totalEvents.to_i)
     end
+
+    it "processes the oldest records first" do
+      # Since Gollum has no stats it will be the first one processed. 
+      # Followed by Frodo and Bilbo.
+      sorted_ids = Sufia::UserStatImporter.new.users_sorted.map {|u| u.id}
```

Comment:
Minor points:
1. I'd expect the method to be named `sorted_users` rather than `users_sorted`.
1. Add spacing inside the braces: `.map { |u| u.id }`